spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestViews.java

```diff
@@ -562,6 +562,58 @@ public void readFromViewWithCTE() throws NoSuchTableException {
     assertThat(sql("SELECT * FROM %s", viewName)).hasSize(1).containsExactly(row(10, 1L));
   }
 
+  @TestTemplate
+  public void readFromViewWithGroupByOrdinal() throws NoSuchTableException {
```

Comment:
please also add a test where the view is entirely created through SQL:

```
@TestTemplate
  public void createViewWithGroupByOrdinal() throws NoSuchTableException {
    insertRows(10);
    String viewName = viewName("createViewWithGroupByOrdinal");
    sql("CREATE VIEW %s AS SELECT id FROM %s GROUP BY 1", viewName, tableName);
    List<Object[]> expected =
        IntStream.rangeClosed(1, 10).mapToObj(this::row).collect(Collectors.toList());

    assertThat(sql("SELECT * FROM %s", viewName))
        .hasSize(10)
        .containsExactlyInAnyOrderElementsOf(expected);
  }
```

Reply:
Thanks for your suggestion, modified.